app/views/tracks.js

```diff
@@ -1,38 +1,49 @@
 import Mn from 'backbone.marionette';
-import Tracks from '../collections/tracks';
 import TrackView from './track';
+import application from '../application';
 
-const TracksView = Mn.CollectionView.extend({  
-    el: '#app-hook',
-    tagName: 'ul',
+
+const TracksView = Mn.CompositeView.extend({
+
+    template: require('./templates/tracks'),
+
+    childViewContainer: '#track-list',
 
     childView: TrackView,
-    events: {
-        'click a': 'play',
-        'click .delete': 'delete'
+
+    onRender() {
+        this.setCurrentTrack();
+        this.listenTo(
+            application.appState,
+            'change:currentTrack',
+            this.setCurrentTrack
+        );
     },
-    play: function (e) {
-        const id = e.currentTarget.dataset.id;
-        const item = this.collection.get(id);
-        item.getStreamURL(playAudio);
+
+    onDestroy() {
+        try {
+            this.stopListening(); // throw weird errors
```

Comment:
https://github.com/cozy-labs/cozy-music/pull/3/files#diff-6a3ff9bec0a7354a982b03fc908248eaR16
I don't understant this bug at all ...
when the view is removed (when we change the current tracks view), it call stoplistening in `remove`, i overrided it in orderd to call it in `ondestroy` and try catch it. Otherwise it break all the app.